hw3/hw3-makinunal/hw3-akinunal.js

```diff
@@ -0,0 +1,131 @@
+/* 1. write a constructor function PersonFunc which accepts the person data like in file people.data.js and implements next methods:
+getFullName() => returns string in format "[FirstName] [LastName]", e.g. 'Viktoriia Chernyshuk';
+getAge() => returns age of person in format "[number] years", e.g. "25 years";
+getFullAddress() => returns address string in format "[country], [city], [street], [house]/[apartment]". Field apartment is optional. E.g. "Ukraine, Lviv, Doroshenka street, 5/8" and "Ukraine, Lviv, Muchna street, 7".
+Please implement error handling in case if some fields are not available.
+*/
+import { SarahMay, JeremieBrown, Elizabeth } from './people.data.js';
+
+console.log('------ QUESTION: 1 ------')
+
+function PersonFunc(prs) {
+    this.person = prs
+    this.fullName = function () {
+        if (!this.person.firstName || !this.person.lastName) {
+            return `Unknown first/last name.`
+        }
+        return `${this.person.firstName} ${this.person.lastName}`;
+    }
+
+    this.getAge = function () {
+        let date = new Date()
```

Comment:
redundant. you could do it with only one variable: `const currentYear = new Date().getFullYear()`. Also if you don't plan to change the variable in future better to use `const`